### PR TITLE
Codify qt qml version in one header

### PR DESCRIPTION
**Pull request by zzmp**
- Only set Qt GL version in one function

### CODE REVIEW

**Comment by zzmp** [comment]
@jherico this is something I noticed while looking into AMD issues. I'm not sure if 4.5 is what we want, but it is what we had, and I thought it would be good to have it only set from one place.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 85b663eb59264f0a974688265f9a78522fa53be1 available at: https://gist.github.com/b0fced9be3cae472a4c00cf54451ae1d<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7610.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7610.dmg

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -1007,6 +1007,9 @@ Application::Application(int& argc, char** argv, QElapsedTimer& startupTimer) :
                 RenderableWebEntityItem* webEntity = dynamic_cast<RenderableWebEntityItem*>(entity.get());
                 if (webEntity) {
                     webEntity->setProxyWindow(_window->windowHandle());
+                    if (Menu::getInstance()->isOptionChecked(KeyboardMouseDevice::NAME)) {
```
Comment (ZappoMan) [inline interface/src/Application.cpp]: how did this get into this PR?

**Comment by hifi-gustavo** [comment]
Build console output for commit: be489e872834c112a0f2027bc3ef242f7b34fdb9 available at: https://gist.github.com/b0fced9be3cae472a4c00cf54451ae1d<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Alpha-PR7610.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Alpha-PR7610.dmg

**Comment by kevinmthomas-carpool** [comment]
## QA Test
**PR Build:** 7610
**Testing Architecture:** 64-bit PC Desktop
**Testing OS:** Windows 10
**Testing Locations:** sandbox, playa, cellscience, forum
**Smoke Test Results:** successful

                                  | playa   | cellscience | forum   
----------------------------------|---------|-------------|--------
Render Rate:                      |61       |64           |62           
Present Rate:                     |60       |60           |60           
Simrate:                          |62       |61           |61            
Render Stats Num Buffers:         |5106     |2520         |4114              
Render Stats gpu::Buffer Memory:  |111.6Mb  |47.5Mb       |72.4Mb              
Render Stats Num Textures:        |826      |213          |901              
Render Stats gpu::Texture Memory: |1320.2Mb |477.0Mb      |879.0Mb              
Render Stats Triangles:           |1472k    |573k         |1364k            
Render Stats Drawcalls:           |1041     |5106         |2759             
Render Stats Items:               |252      |807          |420             

**sandbox ‘createtestblocks.js’ Result:** failure, blocks behaved in an unexpected state as if there was a constant physical pressure against them causing them to move rapidly and unexpectedly without the contact of an avatar
**sandbox Import FBX Result:** successful
**sandbox ‘harmonicOscillator.js’ Result:** successful
**sandbox Echo Server Audio Result:** successful
**sandbox ‘reverbTest.js’ Result:** successful
**Additional Test Results Requested:** (none requested)
**Successful Logout/Crash (If  A Crash Occurs, Attach ‘hifi-log.txt’):** successful

**Pull request merged**